toolkit/offchain/src/reserve/update_settings.rs

```diff
@@ -45,21 +45,21 @@ pub async fn update_reserve_settings<
 	let ctx = TransactionContext::for_payment_key(payment_key, client).await?;
 	let governance = get_governance_data(genesis_utxo, client).await?;
 	let reserve = ReserveData::get(genesis_utxo, &ctx, client).await?;
-	let ReserveUtxo { reserve_utxo, mut reserve_settings } =
+	let ReserveUtxo { reserve_utxo, reserve_settings: mut reserve_datum } =
```

Comment:
It could be just `datum` and `utxo` because Reserve is already present in type name